src/annotator/anchoring/pdf.js

```diff
@@ -448,55 +480,51 @@ export function anchor(root, selectors) {
  * @param {HTMLElement} root - The root element
  * @param {Range} range
  */
-export function describe(root, range) {
-  const normalizedRange = new BrowserRange(range).normalize();
+export async function describe(root, range) {
```

Comment:
Any chance we could, maybe in the future, rename this function? Or does `describe` follow an interface/otherwise have some sort of meaning? ...Ah, I see that it's used by other anchoring classes. Hermp. It's vague.

Reply:
Maybe the `describe` and `anchor` functions could be named something `generateSelectors` and `anchorSelectors` in future?